src/hooks/useIndexOfLastVisibleChild.tsx

```diff
@@ -5,25 +5,32 @@ import { useLayoutEffect, useState } from 'react';
  * that fits within its bounding rectangle. This is done by summing the widths
  * of the children until they exceed the width of the container.
  *
- * @param {Element} containerElementRef - container element
- * @param {Element} overflowElementRef - overflow element
- *
  * The hook returns the index of the last visible child.
+ *
+ * @param containerElementRef - container element
+ * @param overflowElementRef - overflow element
  */
-const useIndexOfLastVisibleChild = (containerElementRef, overflowElementRef) => {
+const useIndexOfLastVisibleChild = (
+  containerElementRef: Element | null,
+  overflowElementRef: Element | null,
```

Comment:
nit/clarification: should these `Element` be `HTMLElement`?

Reply:
Yeah - done in a3fea9e291c5b3fc28ccd4be154d5d5695cd8009